### PR TITLE
[TASK] Update hints of Extension Builder

### PR DESCRIPTION
**Pull request by alexander-nitsche**
See TYPO3-Documentation/T3DocTeam#174 for details.

### CODE REVIEW

File: Documentation/Settings.cfg [hunk 1]
```diff
@@ -1,41 +1,16 @@
 # coding: utf-8
 
-# #####
```
Comment (linawolf) [inline Documentation/Settings.cfg:3]: Changes like these are out of Scope of the PR

**Review by linawolf — CHANGES_REQUESTED** [review]

File: Documentation/Settings.cfg (hunk 1 above)
Comment (alexander-nitsche) [reply Documentation/Settings.cfg:3]: A separate commit should be enough, not?

Comment (linawolf) [reply Documentation/Settings.cfg:3]: No, because the PR will be squash merged

Comment (alexander-nitsche) [reply Documentation/Settings.cfg:3]: Yes. If you like having a squash merge, there will be a PR number referencing to this PR with both commits. And if you like to have cleaning ups separated from the task even in the git log, then let us merge this PR by simply merging it as-is.

**Review by linawolf — APPROVED** [review]
